### DISCUSSION TITLE
starting RPM

### DISCUSSION THREAD

**Discussion by w-i-z-a-r-d** [theOehrly/Fast-F1#127] [Q&A] [answered]
Hello,
is it possible to get the telemetry from couple of seconds before the start before the red lights are off ?

**Comment by theOehrly** [comment, marked as answer]
Yes, for a race, you should be able to extend the data of the first lap to include a few more samples in the beginning. For example

```
lec_first = session.laps.pick_driver('LEC').iloc[0]
lec_car_data = lec_first.get_car_data(pad=10, pad_side='before')
```

The `pad` value is the number of samples to use for padding.

You can also access each driver's full car and position data through the dictionaries `Session.car_data` and `Session.pos_data`. An then use  `Telemetry.slice_by_time` for example, to get the section of car data that you are looking for.
